testing/yodl/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=yodl
+pkgver=3.08.02
+pkgrel=0
+pkgdesc="a pre-document language and tools to process it"
+url="https://fbb-git.github.io/yodl/"
+arch="all"
+license="GPL2"
+makedepends="icmake bash perl flex bison"
+subpackages="$pkgname-doc"
+source="yodl-$pkgver.tar.gz::https://github.com/fbb-git/yodl/archive/$pkgver.tar.gz"
+builddir="$srcdir/yodl-$pkgver/yodl"
+options="!check"
```

Comment:
If upstream does not provide any tests and the project is an application (not a library), then it’s better to add at least very basic check that just calls the produced binary with `--version`. Just to check that the produced binary is not totally broken.